collect_app/src/main/java/org/odk/collect/android/dao/InstancesDao.java

```diff
@@ -61,24 +64,27 @@ public CursorLoader getSentInstancesCursorLoader(CharSequence charSequence, Stri
         return cursorLoader;
     }
 
-    public CursorLoader getSentInstancesCursorLoader(String sortOrder) {
-        String selection = InstanceProviderAPI.InstanceColumns.STATUS + " =? ";
-        String[] selectionArgs = {InstanceProviderAPI.STATUS_SUBMITTED};
+    private CursorLoader getSentInstancesCursorLoader(String sortOrder) {
+        String selection = InstanceProviderAPI.InstanceColumns.STATUS + "=? or "
+                + InstanceProviderAPI.InstanceColumns.STATUS + "=?";
+        String[] selectionArgs = {InstanceProviderAPI.STATUS_SUBMITTED, InstanceProviderAPI.STATUS_SUBMITTED_SMS};
 
         return getInstancesCursorLoader(null, selection, selectionArgs, sortOrder);
     }
 
     public Cursor getUnsentInstancesCursor() {
-        String selection = InstanceProviderAPI.InstanceColumns.STATUS + " !=? ";
-        String[] selectionArgs = {InstanceProviderAPI.STATUS_SUBMITTED};
+        String selection = InstanceProviderAPI.InstanceColumns.STATUS + "!=? or "
```

Comment:
I think it should be `and` not `or` here.

Reply:
Oh, I didn't realize this was the unsent case. Yes, you're absolutely correct, @grzesiek2010. https://en.wikipedia.org/wiki/De_Morgan%27s_laws

Reply:
Oh yes! Good catch @grzesiek2010